src/components/notification/process-server-status.ts

```diff
@@ -67,44 +67,48 @@ const determineProbeState = ({
   recoveryThreshold,
 }: {
   errorName: string
-  probeStatus: StatusDetails
+  probeStatus: ProbeStateDetails
   validation: ValidateResponse
   incidentThreshold: number
   recoveryThreshold: number
 }) => {
   const { isDown, consecutiveTrue, consecutiveFalse } = probeStatus
-  const { status } = validation
+  const { somethingToReport } = validation
```

Comment:
Will `hasSomethingToReport` become better variable name for this one?

Reply:
i thought of it.. but wanted to keep it short too. I think without it and being boolean is quite clear. let me know if anyone disagrees.